### PR TITLE
add rongcloud web sdk

### PR DESCRIPTION
**Pull request by liusong1111**
add rongcloud web sdk:

https://github.com/rongcloud/demo-web-sdk

rongcloud is a chinese instant messaging cloud service.

I use it in my personal project, and make its extern file to work with cljs.

### CODE REVIEW

File: rongcloud/build.boot [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  (comp
+    ;(download :url "https://raw.githubusercontent.com/rongcloud/rongcloud-web-im-sdk/master/dest/RongIMClient.js"
+    ;          :checksum "5493CB844C0A73BB752EBB422E11547A"
+    ;          :unzip false)
+    ;(download :url "https://raw.githubusercontent.com/rongcloud/rongcloud-web-im-sdk/master/dest/RongIMClient.min.js"
+    ;          :checksum "A4C42981A728720DAE5E532270D77B05"
+    ;          :unzip false)
+    ;(sift :move {#"^RongIMClient.js-\d\.\d\.\d/RongIMClient.js"     "cljsjs/development/rongcloud.inc.js"
+    ;             #"^RongIMClient.js-\d\.\d\.\d/RongIMClient.min.js" "cljsjs/production/rongcloud.min.inc.js"})
+    ;(replace-content :in "cljsjs/production/rongcloud.min.inc.js" :match #"(?m)^//# sourceMappingURL=.*$" :value "")
```
Comment (martinklepsch) [inline rongcloud/build.boot]: Should the lines above be commented out?

**Comment by Deraen** [comment]
Would be much preferred if the source code wasn't included in this repo.

**Comment by liusong1111** [comment]
Sorry for late reply. 
I recommend ignore this Pull Request, because rongcloud is a CLOUD service which is improved constantly. 
It's better to be supported by rongcloud Inc?  :)

**Comment by martinklepsch** [comment]
@liusong1111 that would require Rongcloud to provide a jar with proper externs. I doubt they will do that. When the JS is updated and you need the updated version you can usually just change a few version numbers and open a PR. As @Deraen said, for this to be possible you need to use the kind of code thats commented out in your `build.boot` and not commit the actual source files.

**Comment by Deraen** [comment]
Closing as inactive.

**Pull request closed without merging**